csrankings-4.csv

```diff
@@ -338,6 +338,7 @@ Jihad Mohamad Jaam,Qatar University,http://qspace.qu.edu.qa/handle/10576/61/disc
 Jihong Kim 0001,Seoul National University,http://cares.snu.ac.kr,E9A5Rv8AAAAJ
 Jihong Zhu,Tsinghua University,http://www.cs.tsinghua.edu.cn/publish/csen/4923/2014/20140108153546275916040/20140108153546275916040_.html,3K-zt_oAAAAJ
 Jihun Hamm,Tulane University,http://www.cs.tulane.edu/~jhamm3,DdRsMKsAAAAJ
+Jiji Zhang,Hong Kong Baptist University,https://rel.hkbu.edu.hk/people/zhang-jiji,5GAikocAAAAJ

```

Comment:
Not in CS (Religion & Philosophy).

Reply:
While being another department, his recent research area is heavily focused on machine learning and AI. You can see  https://scholar.google.com/citations?user=5GAikocAAAAJ&hl=en for more info.

Reply:
I have removed this entry.